packages/react-reconciler/src/__tests__/SchedulingProfiler-test.internal.js

```diff
@@ -158,16 +159,33 @@ describe('SchedulingProfiler', () => {
       return <Bar />;
     }
 
-    ReactNoop.render(<Foo />);
-    // Do one step of work.
-    expect(ReactNoop.flushNextYield()).toEqual(['Foo']);
-
-    expectMarksToEqual([
-      `--react-init-${ReactVersion}`,
-      `--schedule-render-${formatLanes(ReactFiberLane.DefaultLane)}`,
-      `--render-start-${formatLanes(ReactFiberLane.DefaultLane)}`,
-      '--render-yield',
-    ]);
+    if (gate(flags => flags.enableSyncDefaultUpdates)) {
+      React.unstable_startTransition(() => {
+        ReactNoop.render(<Foo />);
+      });
+
+      // Do one step of work.
+      expect(ReactNoop.flushNextYield()).toEqual(['Foo']);
+
+      expectMarksToEqual([
+        `--react-init-${ReactVersion}`,
+        `--schedule-render-${formatLanes(ReactFiberLane.TransitionLane1)}`,
```

Comment:
I know the import was already there, but can we please hardcode these instead of importing from a module? Doesn't work with bundle tests and makes it too tempting for that export to leak elsewhere.

Reply:
Done